app/assets/javascripts/admin_users.js

```diff
@@ -43,10 +43,15 @@
 
   // Hide password fields when non-internal authentication source is selected
   function on_auth_source_change() {
+    var password_fields = jQuery('#password_fields');
+        password_inputs = password_fields.find('#user_password, #user_password_confirmation');
+
     if (this.value === '') {
-      jQuery('#password_fields').show();
+      password_fields.show();
+      password_inputs.removeAttr('disabled');
     } else {
-      jQuery('#password_fields').hide();
+      password_fields.hide();
+      password_inputs.prop('disabled', 'disabled');
```

Comment:
Identifier 'password_inputs' is not in camel case.<br>'password_inputs' is not defined.